src/metrics.cc

```diff
@@ -112,18 +95,23 @@ void Metrics::Report() {
   for (vector<Metric*>::iterator i = metrics_.begin();
        i != metrics_.end(); ++i) {
     Metric* metric = *i;
-    double total = metric->sum / (double)1000;
-    double avg = metric->sum / (double)metric->count;
+    uint64_t micros = TimerToMicros(metric->sum);
+    double total = micros / (double)1000;
+    double avg = micros / (double)metric->count;
     printf("%-*s\t%-6d\t%-8.1f\t%.1f\n", width, metric->name.c_str(),
            metric->count, avg, total);
   }
 }
 
-uint64_t Stopwatch::Now() const {
-  return TimerToMicros(HighResTimer());
+double Stopwatch::Elapsed() const {
+  // Convert to micros after converting to double to minimize error.
+  return 1e-6 * TimerToMicros(static_cast<double>(NowRaw() - started_));
```

Comment:
1e-6 is a real thing? I've never seen this syntax.

Reply:
Yep. 1 times 10 to the -6 power. It's standard exponential notation. I guess you can never tell what will be recognizable or not to other software developers.

Any thoughts on how to enable C++11 on Ubuntu 2016.04?

Reply:
Sorry, I didn't see this before.

> Any thoughts on how to enable C++11 on Ubuntu 2016.04?

You're referring to the CI build instructions? You'd have to modify .github/workflows/* and CMakeLists.txt

But I think you've already answered your question, give you created https://github.com/ninja-build/ninja/pull/2089